Example/PaymentSheet Example/PaymentSheet Example/PaymentSheetTestPlayground.swift

```diff
@@ -633,6 +636,35 @@ extension PaymentSheetTestPlayground: EndpointSelectorViewControllerDelegate {
     }
 }
 
+extension PaymentSheetTestPlayground {
```

Comment:
In the meantime, let me know if you have any better ideas on confirming if the callback was invoked.

Reply:
Seems fine to me